### ISSUE TITLE
Recipe parameters for variables?

### ISSUE THREAD

**Issue by bukowa** [casey/just#2209]
Can it be done?

```justfile
[unix]
export VAR=1

[windows]
export VAR=2
```

**Comment by laniakea64** [comment]
So you're seeking to export different values for a particular environment variable depending whether running on Windows or a Unix OS?

Does this do what you're looking for? -
```
export VAR := if os_family() == 'windows' { "2" } else { "1" }
```

**Comment by bukowa** [comment]
> So you're seeking to export different values for a particular environment variable depending whether running on Windows or a Unix OS?
> 
> Does this do what you're looking for? -
> 
> ```
> export VAR := if os_family() == 'windows' { "2" } else { "1" }
> ```

Yes, thank you. I was hoping for a (e.g. undocumented) more expressive expression like in my example.

**Comment by casey** [comment]
I'm not opposed to allowing OS attributes on variables, so I'll leave this open.

**Comment by casey** [comment]
Implemented in #3515!
